src/bin/vip-wp.js

```diff
@@ -376,7 +403,7 @@ commandWrapper( {
 			if ( ( empty || ! startsWithWp ) && ! userCmdCancelled ) {
 				console.log(
 					chalk.red( 'Error:' ),
-					'invalid command, please pass a valid WP CLI command.'
+					'invalid command, please pass a valid WP-CLI command.'
```

Comment:
```suggestion
					'Invalid command, please pass a valid WP-CLI command.'
```

There are inconsistencies in the codebase about how error responses are displayed (e.g., some have a [red "x Error:" prefix](https://github.com/Automattic/vip-cli/blob/b1509e1cdfb1a5bec3469c70331ebc65bc0d8c93/src/bin/vip-import-validate-files.js#L77), and some [have a red "x" prefix before Error](https://github.com/Automattic/vip-cli/blob/b1509e1cdfb1a5bec3469c70331ebc65bc0d8c93/src/lib/vip-import-validate-files.ts#L300)) but it seems like there is some precedent for sentence casing the message

https://github.com/Automattic/vip-cli/blob/b1509e1cdfb1a5bec3469c70331ebc65bc0d8c93/src/lib/vip-import-validate-files.ts#L301

Reply:
Yeah, as much as I want to do a thorough grooming of all error messages I don't think I can handle that level of scope creep. I probably should have exercised more restraint and not touched the error messages at all, but here we are.